contrib/forms/assessment_intake/view.php

```diff
@@ -376,4 +394,4 @@
 </form>
 <?php
 formFooter();
-?>
+?> 
```

Comment:
This seems odd, since whitespace at end of php scripts can sometime break things. Usually either need to ensure no whitespace (like it was before the change or simply remove the closing tag altogether.